### PR TITLE
Remove katello-agent related entities

### PR DESCRIPTION
**Pull request by vsedmik**
##### Description of changes

Katello-agent was deprecated and removed from 6.15 including related API endpoints. This PR removes those supported in nailgun.

##### Upstream API documentation, plugin, or feature links

See the 6.15 apidoc for these particular endpoint. They are missing.

- PUT /api/hosts/:host_id/errata/apply
- PUT /api/hosts/:host_id/packages/install 
- PUT /api/hosts/:host_id/packages/upgrade
- PUT /api/hosts/:host_id/packages/upgrade_all
- PUT /api/hosts/:host_id/packages/remove
- PUT /api/hosts/bulk/install_content
- PUT /api/hosts/bulk/update_content
- PUT /api/hosts/bulk/remove_content 

##### Functional demonstration

```
In [1]: from robottelo.hosts import Satellite
In [2]: sat = Satellite('<SAT_FQDN>')	# stream/6.15 sat
In [3]: sat.api.Host().install_content(data={'organization_id': '1', 'included': {'ids': ['1']}, 'content_type': 'package', 'content': ['cat']})
...
HTTPError: 404 Client Error: Not Found for url: https://<SAT_FQDN>/api/v2/hosts/bulk/install_content
```

##### Additional Information

Robottelo PR for removal of the last used endpoint (`/api/hosts/bulk/install_content`) - https://github.com/SatelliteQE/robottelo/pull/12817

### CODE REVIEW

**Review by synkd — APPROVED** [review]
ACK

**Comment by vsedmik** [comment]
rebased

File: nailgun/entities.py [hunk 1]
```diff
@@ -4378,44 +4378,6 @@ def errata_applicability(self, synchronous=True, timeout=None, **kwargs):
         response = client.put(self.path('errata/applicability'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
-    def errata_apply(self, synchronous=True, timeout=None, **kwargs):
```
Comment (Gauravtalreja1) [inline nailgun/entities.py:4381]: Can we remove errata_applicability method too as it is also removed right?

**Review by damoore044 — APPROVED** [review]
Ack pending other comment

File: nailgun/entities.py (hunk 1 above)
Comment (vsedmik) [reply nailgun/entities.py:4424]: Negative. Errata applicability recalculation functionality remains present.
```
[GET /api/hosts/:host_id/errata]    List errata available for the content host
[GET /api/hosts/:host_id/errata/:id]    Retrieve a single errata for a host
[PUT /api/hosts/:host_id/errata/applicability]    Force regenerate applicability.
```

**Review by sambible — APPROVED** [review]

**Pull request merged**